docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 45
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Mimir through the Prometheus data source type.
+
+## Before you begin
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. In the **HTTP** > **Prometheus server URL** field, point to the proxy in front of Mimir (such as Nginx or the GEM gateway) rather than to one component of Mimir.
+
+   - When running locally or with Docker using port mapping,
+     the address is likely `http://localhost:9090`.
+
+   - When running with docker-compose or Kubernetes,
+     the address is likely `http://prometheus:9090`.
+
+## Steps
+
+1. To see metrics, select **Explore**.
```

Comment:
If the user has already completed the "before you begin" steps, they're going to skip right down to this part.  Maybe update this to provide context for where they should be.
```suggestion
1. To view metrics in Grafana, select **Explore**.
```